moveit_core/robot_model/src/robot_model.cpp

```diff
@@ -295,7 +295,7 @@ void RobotModel::buildJointInfo()
     }
   }
 
-  bool link_considered[link_model_vector_.size()] = { false };
+  std::vector<bool> link_considered(link_model_vector_.size(), false);
```

Comment:
Thank you clang! I wonder if this ever broke before...

Reply:
``std::vector<bool>`` can be an unweildy vector, are you sure this is safe? https://isocpp.org/blog/2012/11/on-vectorbool

Reply:
std::vector<bool> is safe. The reference you mentioning, is only complaining about the name of the data structure (and potentially misleading compiler errors when this non-perfect container is used as a normal container). We don't have this issue here.